### PR TITLE
Patch 5

### PR DESCRIPTION
**Pull request by andruwik777**
please review and merge

### CODE REVIEW

File: file3.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+file3.txt from 777 init
```
Comment (andruwik777uafm) [inline file3.txt:1]: wtf?

Comment (andruwik777) [inline file3.txt:1]: whats wrong?

Comment (andruwik777uafm) [inline file3.txt:1]: this is trash

Comment (andruwik777uafm) [inline file3.txt:1]: ttttrrraaassshhhh

**Comment by andruwik777uafm** [comment]
goood

**Pull request merged**